### DISCUSSION TITLE
Welcome to miru-web Discussions!

### DISCUSSION THREAD

**Discussion by akhilgkrishnan** [saeloun/miru-web#676] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vandelin586** [comment]
When I run this project locally on ubuntu WSL, everything works, well I should say I am prompted with a logon screen at 127.0.0.1:3000

Just wondering the default logins? 

I have tried 
Role    Email    Password
Owner    vipul@example.com    welcome
Admin    supriya@example.com    welcome
Employee    sam@example.com    welcome
Client    oliver@example.com    welcome

But none of these accounts are able to login while running locally. I mean I have an account at https://app.miru.so/ but I would like to run this all locally.... perhaps I do not understand this project fully... Is this possible to have it all running locally including the user accounts?

**Reply by akhilgkrishnan** [reply]
Could you please run the `rails db:seed`

**Reply by vandelin586** [reply]
Thank you for this response !!! I will give this a try tonight

**Reply by vandelin586** [reply]
This worked, thank you @akhilgkrishnan

**Reply by akhilgkrishnan** [reply]
Nice 👍
